### PR TITLE
Update credits for 5.8.0 release

### PR DESCRIPTION
**Pull request by sfan5**

### CODE REVIEW

File: builtin/mainmenu/tab_about.lua [hunk 1]
```diff
@@ -28,8 +28,9 @@ local core_developers = {
 	"Lars Hofhansl <larsh@apache.org>",
 	"v-rob <robinsonvincent89@gmail.com>",
 	"Hugues Ross <hugues.ross@gmail.com>",
```
Comment (Desour) [inline builtin/mainmenu/tab_about.lua:30]: Also left us I think: https://github.com/orgs/minetest/discussions/59

Comment (sfan5) [reply builtin/mainmenu/tab_about.lua]: Oh, indeed.

**Comment by Bituvo** [comment]
Thanks for adding me :O

File: builtin/mainmenu/tab_about.lua [hunk 2]
```diff
@@ -43,18 +43,19 @@ local core_team = {
...
-	"Lexi Hale [Particlespawner animation]",
+	"Wuzzy [Features, translations, documentation]",
+	"numzero [Optimizations, work on OpenGL driver]",
+	"ROllerozxa [Bugfixes, Mainmenu]",
+	"Lars Müller [Bugfixes]",
+	"AFCMS [Documentation]",
 	"savilli [Bugfixes]",
 	"fluxionary [Bugfixes]",
-	"Gregor Parzefall [Bugfixes]",
+	"Thresher [Documentation]",
```
Comment (Bituvo) [inline builtin/mainmenu/tab_about.lua:53]: Please update my name.

Comment (sfan5) [reply builtin/mainmenu/tab_about.lua:53]: `Bradley Pierce` or `Bradley Pierce (Thresher)` or `Bradley Pierce (Bituvo)`?

Comment (Bituvo) [reply builtin/mainmenu/tab_about.lua:53]: `Bradley Pierce (Thresher)` please. I chose the name Bituvo a while ago and I've grown old of it...

File: builtin/mainmenu/tab_about.lua [hunk 3]
```diff
@@ -27,9 +27,9 @@ local core_developers = {
 	"Krock/SmallJoker <mk939@ymail.com>",
 	"Lars Hofhansl <larsh@apache.org>",
 	"v-rob <robinsonvincent89@gmail.com>",
-	"Hugues Ross <hugues.ross@gmail.com>",
-	"Dmitry Kostenko (x2048) <codeforsmile@gmail.com>",
 	"Desour",
```
Comment (Desour) [inline builtin/mainmenu/tab_about.lua:30]: While you're at it, could you please add my short name?
```suggestion
	"Desour/DS",
```

**Review by rubenwardy — APPROVED** [review]

**Pull request merged**